CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Added
+
+- dependency on [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler) binary
```

Comment:
A bit more description what it does would be nice - forge being independent of sierra version, why is there another binary (independent updates) etc.

Maybe a blog post about it would be nice? A tweet/tg message is a must imo.

I heard starkli wants to use USC as well so this is a thing we can use to advertise ourselves xD